eras/babbage/impl/src/Cardano/Ledger/Babbage/TxBody.hs

```diff
@@ -245,6 +245,14 @@ data Datum era
   | Datum !(BinaryData era)
   deriving (Eq, Ord, Show)
 
+instance Semigroup (Datum era) where -- This instance is left biased
```

Comment:
was this instance needed?

Reply:
Yes it is used in merging in the generic stuff.

Reply:
Not true any more, with the simplified merging. So I removed it.